float32 lambda
---
bool success
string message
